boards/ct/ctcc/doc/index.rst

```diff
@@ -0,0 +1,195 @@
+.. _ctcc_nrf52840:
+
+CTHINGS.CO Connectivity Card nRF52840
+#####################################
+
+Overview
+********
+
+The Connectivity Card nRF52840 enables BLE and IEEE 802.15.4 connectivity
+over mPCIe or M.2 using USB port with on-board nRF52840 SoC.
+
+This board has following features:
+
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* RADIO (Bluetooth Low Energy and 802.15.4)
+* :abbr:`RTC (nRF RTC System Clock)`
+* :abbr:`USB (Universal Serial Bus)`
+* :abbr:`WDT (Watchdog Timer)`
+
+.. figure:: img/ctcc_nrf52840.png
+     :align: center
+     :width: 300
+     :alt: ctcc_nrf52840
+
+     ctcc_nrf52840 board
+
+More information about the board can be found at the
+`ctcc_nrf52840 Website`_ and for SoC information: `Nordic Semiconductor Infocenter`_.
+
+Hardware
+********
+
+The ``ctcc_nrf52840`` has one external oscillator of the 32.768 kHz.
+
+Supported Features
+==================
+
+The ``ctcc_nrf52840`` board configuration supports the following
```

Comment:
these are board targets (apply above comment also), board configuration does not exist anymore. Update all instances